tools/build_all.sh

```diff
@@ -25,13 +18,8 @@ set -e
 
 CWD=`pwd`
 
-# Determine which repos to build. If the ROS 1 and ROS 2 base directories are
+# Determine which repos to build. If ROS 2 base directories are
```

Comment:
I assume there is only one ROS2 base directory:

"If ROS2 base directories are not present..." -> "If the ROS2 base directory is not present..."